pythontest.py

```diff
@@ -4,11 +4,14 @@
 file1 = '/Users/johannesschmid/Desktop/CodeBaking/Python Project/sheet1.xlsx'
 file2 = '/Users/johannesschmid/Desktop/CodeBaking/Python Project/sheet2.xlsx'
 
+def getExtension(file):
+    return os.path.splitext(file)[1]
+
 
 def formatcheck(file):
     # Splitting path and file extension off
     filepath = os.path.dirname(file)
-    file_extension = os.path.splitext(file)[1]
+    file_extension = getExtension(file)
 
     # Changing working directory to filepath
     os.chdir(filepath)
```

Comment:
Let's try to avoid this, it complicates stuff. Let's just pass the file name instead of the whole path and assume they're always in the same folder as the script.
We can look together at the environments settings together if that's still going wrong.